designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/StatisticUrlConfig.scala

```diff
@@ -9,5 +9,5 @@ final case class StatisticUrlConfig(
     nuStatsUrl: String = "https://stats.nussknacker.io/?",
     queryParamsSeparator: String = "&",
     emptyString: String = "",
-    maybePublicEncryptionKey: Option[PublicEncryptionKey] = None
+    publicEncryptionKey: PublicEncryptionKey = PublicEncryptionKey.INSTANCE
```

Comment:
We should remove `PublicEncryptionKey.INSTANCE` because we have the same key in two places now.